components/date-picker/picker.component.ts

```diff
@@ -135,6 +139,7 @@ export class NzPickerComponent implements OnInit, AfterViewInit {
     if (this.realOpenState) {
       this.overlayOpen = false;
       this.openChange.emit(this.overlayOpen);
+      this.focus();
```

Comment:
Need a test for `focus`.

Reply:
Added 'should focus on the trigger after a click outside' tests

Reply:
![image](https://user-images.githubusercontent.com/53300025/77054891-28ce1380-6a0b-11ea-9bd7-5189264a3476.png)
is this function implemented? The pop-up date box automatically captures the focus inside and can be entered by the keyboard